scripts/release-afterdoc.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+main() {
+  local -r RELNAME="$1"
+  local -r RELEASE="$(curl -sSL -H "Accept: application/vnd.github.v3+json" "https://api.github.com/repos/kube-rs/kube-rs/releases/tags/${RELNAME}")"
+  local -r HURL="$(echo "${RELEASE}" | jq '.html_url' -r)"
+  # Skipping New Contributors highight from CHANGELOG + across repos for brevity and to avoid pinging them excessively
+  local -r BODY="$(echo "${RELEASE}" | jq '.body' -r | sd "## New Contributors[\w\W]*$" "")"
+  if grep -E "^${RELNAME} / " CHANGELOG.md; then
+    # We only run the script if the headline is unchanged (done at the end)
+
+    # Add in the body first
+    sd "(^${RELNAME} / [\d-]+\n===================\n)" "\$1${BODY}" CHANGELOG.md
```

Comment:
ugh.. yeah, possibly. it never double matched for me because it would have to match something weird like 0a68c0 which has a really small chance of occurring in the document (even less so at the beginning of a line followed by a slash)..

can maybe fix it with a quick bash replace tho :thinking:

Reply:
nope, but managed to do it with a smaller `sd`, and moved the entire script to just use `sd` (no `sed`)

Reply:
Looks like there are still a few cases of `$RELNAME` in regexes

Reply:
got it now!